src/realm/sync/noinst/client_reset_operation.cpp

```diff
@@ -0,0 +1,61 @@
+#include <realm/db.hpp>
+#include <realm/sync/encrypt/fingerprint.hpp>
+#include <realm/sync/history.hpp>
+#include <realm/sync/noinst/common_dir.hpp>
+#include <realm/sync/noinst/compression.hpp>
+#include <realm/sync/noinst/client_history_impl.hpp>
+#include <realm/sync/noinst/client_reset.hpp>
+#include <realm/sync/noinst/client_reset_operation.hpp>
+
+using namespace realm;
```

Comment:
Instead of doing `using namespace` for the bodies of implementation files, let's just enclose the implementation in the correct namespace. You can do
```
namespace realm::_impl {
body of cpp file
} // namespace realm::_impl
```

That way we don't risk adding any globals to the top-level global scope.